src/Magento/FunctionalTestingFramework/DataTransport/Auth/WebApiAuth.php

```diff
@@ -63,7 +70,12 @@ public static function getAdminToken($username = null, $password = null)
         }
 
         if (isset(self::$adminAuthTokens[$login])) {
-            return self::$adminAuthTokens[$login];
+            $threeHours = 60 * 60 * 3;
```

Comment:
Can we avoid using hard coded value? What if the value are not default?

Reply:
So you're suggesting to add an option to .env?

Reply:
Yes. a configuration with suggested default value. You can check etc/.env.example.